django_geosource/migrations/0017_auto_20200903_1431.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.1 on 2020-09-03 14:31
+
+import django.core.validators
+from django.db import migrations, models
+import django_geosource.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('django_geosource', '0016_auto_20200506_1444'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='field',
+            name='data_type',
+            field=models.IntegerField(choices=[(1, django_geosource.models.FieldTypes['String']), (2, django_geosource.models.FieldTypes['Integer']), (3, django_geosource.models.FieldTypes['Float']), (4, django_geosource.models.FieldTypes['Boolean']), (5, django_geosource.models.FieldTypes['Undefined']), (6, django_geosource.models.FieldTypes['Date'])], default=5),
```

Comment:
Is default "undefined" ?

Reply:
It looks like, but I don't know if it should be. What do you think?

Reply:
```py
    data_type = models.IntegerField(
        choices=FieldTypes.choices(), default=FieldTypes.Undefined.value
    )
```